java.sql.SQLException: Failed to start database 'derby_dir/name1', see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.newEmbedSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.seeNextException(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.bootDatabase(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection30.<init>(Unknown Source)
	at org.apache.derby.impl.jdbc.EmbedConnection40.<init>(Unknown Source)
	at org.apache.derby.jdbc.Driver40.getNewEmbedConnection(Unknown Source)
	at org.apache.derby.jdbc.InternalDriver.connect(Unknown Source)
	at org.apache.derby.jdbc.AutoloadedDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at spec.benchmarks.derby.DerbyHarness.getNestedConnection(DerbyHarness.java:482)
	at spec.benchmarks.derby.DerbyHarness$Client.<init>(DerbyHarness.java:41)
	at spec.benchmarks.derby.DerbyHarness.<init>(DerbyHarness.java:281)
	at spec.benchmarks.derby.DerbyHarness.main(DerbyHarness.java:248)
	at spec.benchmarks.derby.Main.runBenchmark(Main.java:29)
	at spec.benchmarks.derby.Main.harnessMain(Main.java:143)
	at spec.harness.BenchmarkThread.runLoop(BenchmarkThread.java:170)
	at spec.harness.BenchmarkThread.executeIteration(BenchmarkThread.java:132)
	at spec.harness.BenchmarkThread.run(BenchmarkThread.java:59)
Caused by: java.sql.SQLException: Failed to start database 'derby_dir/name1', see the next exception for details.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	... 21 more
Caused by: java.sql.SQLException: Derby system shutdown.
	at org.apache.derby.impl.jdbc.SQLExceptionFactory.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.wrapArgsForTransportAcrossDRDA(Unknown Source)
	at org.apache.derby.impl.jdbc.SQLExceptionFactory40.getSQLException(Unknown Source)
	at org.apache.derby.impl.jdbc.Util.generateCsSQLException(Unknown Source)
	... 18 more
Caused by: ERROR XJ015: Derby system shutdown.
	at org.apache.derby.iapi.error.StandardException.newException(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.bootService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startProviderService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.findProviderAndStartService(Unknown Source)
	at org.apache.derby.impl.services.monitor.BaseMonitor.startPersistentService(Unknown Source)
	at org.apache.derby.iapi.services.monitor.Monitor.startPersistentService(Unknown Source)
	... 18 more
ERROR: connection == null
java.lang.ArithmeticException: Division undefined
	at java.math.BigDecimal.divide(Unknown Source)
	at spec.benchmarks.derby.DerbyHarness$Client.resultsToString(DerbyHarness.java:58)
	at spec.benchmarks.derby.DerbyHarness.<init>(DerbyHarness.java:283)
	at spec.benchmarks.derby.DerbyHarness.main(DerbyHarness.java:248)
	at spec.benchmarks.derby.Main.runBenchmark(Main.java:29)
	at spec.benchmarks.derby.Main.harnessMain(Main.java:143)
	at spec.harness.BenchmarkThread.runLoop(BenchmarkThread.java:170)
	at spec.harness.BenchmarkThread.executeIteration(BenchmarkThread.java:132)
	at spec.harness.BenchmarkThread.run(BenchmarkThread.java:59)
